libs/core/langchain_core/messages/tool.py

```diff
@@ -195,7 +195,7 @@ class ToolCall(TypedDict):
 
     name: str
     """The name of the tool to be called."""
-    args: dict[str, Any]
+    args: Optional[dict[str, Any]]
```

Comment:
Why is this change necessary? 

An empty dict represents no args currently. I'd rather not add a second way to represent the same information